runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -87,6 +87,7 @@ def __init__(
             domain=domain,
             den_auth=den_auth,
             dryrun=dryrun,
+            creds=kwargs.pop("ssh_creds", None),
```

Comment:
@carolineechen i think we want this? otherwise it's not being used at all by the `Cluster` parent when setting up the ssh creds

Reply:
it's not needed since we don't support custom ssh creds in oss. we only support 1) launching with existing sky-key 2) setting default ssh creds to sky-key or 3) generating sky key during launch

Reply:
ok cool, that makes sense. a sign that we're not doing something right in the launcher then